sig-scalability/README.md

```diff
@@ -161,7 +161,10 @@ more generalized benchmarking scenarios.
 
 Ensuring that all tests necessary to validate Kubernetes scalability and
 performance exists (ideally by providing easy-to-use frameworks and working
-with SIGs to provide them), having environment and resources to run them.
+with SIGs to provide them), having environment and resources to run them:
```

Comment:
move those changes to a separate PR

Reply:
Done https://github.com/kubernetes/community/pull/6503